src/wp-includes/blocks/index.php

```diff
@@ -5,6 +5,10 @@
  * @package WordPress
  */
 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
```

Comment:
```suggestion
// Don't load directly.
if ( ! defined( 'ABSPATH' ) ) {
	die( '-1' );
}
```

Core use above code for checking. For consistency can we use that?

Reply:
@mukeshpanchal27 suggestion updated.